### DISCUSSION TITLE
Lecture review

### DISCUSSION THREAD

**Discussion by petra-Nam** [petra-Nam/learn-java#2] [General]
Today, I finished setting up GitHub CodeSpaces for my project, and I added a Wiki to document my progress
I will blog regularly about my coding experiences feel free to comment and advice where necessary.
Hello World!!!!:))))

**Comment by Roic22** [comment]
Nice to know you could successfully get along with the settings and I hope you really have a good time playing around with the various features this space offers

**Comment by fjkun06** [comment]
Awesome! It's recommended to start small until you get the hang  of it. Just know you already have a fan in me and that programming is all about the errors we make and how we gain experience from them. Feel free to make mistakes.

**Comment by Obrienmaina-Mosbach** [comment]
Hi Petra,

That’s awesome! Setting up CodeSpaces and adding a Wiki is a great way to stay organized. Looking forward to seeing how your project evolves—keep up the great work!

Best,
Brian
